.evergreen/download-mongodb.sh

```diff
@@ -9,6 +9,9 @@ get_distro ()
    if [ -f /etc/os-release ]; then
       . /etc/os-release
       DISTRO="${ID}-${VERSION_ID}"
+   elif [ -f /etc/centos-release ]; then
+      version=$(cat /etc/centos-release | tr -dc '0-9.' | cut -d '.' -f1)
+      DISTRO="centos-${version}"
```

Comment:
Was this only required for `linux-centos-6*`? I assume RHEL is handled by `elif [ -f /etc/redhat-release ];` later on?